components/update_user_group_modal/update_user_group_modal.scss

```diff
@@ -0,0 +1,42 @@
+.modal {
```

Comment:
Same here, I would expect a component name (with upper case) as the root class to get this scoped properly.

Reply:
Root class is now related to the component name